### ISSUE TITLE
KGTK does not work with Inkscape and VSCode on Arch Linux

### ISSUE THREAD

**Issue by gusbemacbe** [sandsmark/kgtk#9]
Hello!

Inkscape:

```
❯ /usr/bin/kgtk-wrapper /usr/bin/inkscape
Timeout: 30
Icon theme "gnome" not found.
New connection
now have 1 connections
new client... "inkscape"  ( 11 )

(inkscape:157044): Gtk-ERROR **: 21:01:44.963: GTK+ 2.x symbols detected. Using GTK+ 2.x and GTK+ 3 in the same process is not supported
read 11
Comms error, closing connection... 11
close 11
Delete client
no connections, starting timer
QSocketNotifier: Invalid socket 11 and type 'Read', disabling...
QSocketNotifier: Invalid socket 11 and type 'Exception', disabling...
Deleted client "inkscape"
zsh: trace trap (core dumped)  /usr/bin/kgtk-wrapper /usr/bin/inkscape
```

And VSCode still opens the GTK2 Dialog.

**Comment by sandsmark** [comment]
I assume vscode just uses the same code as chromium? chromium has really ugly logic for determining the dialog it should use, so not surprised if it doesn't work.

as for inkscape, did you build it against gtk3 or gtk2? inkscape is still gtk2.

**Comment by gusbemacbe** [comment]
> I assume vscode just uses the same code as chromium? chromium has really ugly logic for determining the dialog it should use, so not surprised if it doesn't work.

Yes, I suspect VSCode is based on Chromium 76 and uses still uses Electron 6, which may not work with KDE dialog. My Google Chrome, which is 81, here on Arch Linux, opens the KDE dialog normally. 

> as for inkscape, did you build it against gtk3 or gtk2? inkscape is still gtk2.

I built the beta version of Inkscape from AUr (check the https://aur.archlinux.org/packages/inkscape-git). The PKGBUILD does not tell which GTK version it will be built against, but the error indicate sthat Inkscape is built against GTK2.

**Comment by MountainX** [comment]
The Arch Linux [package](https://aur.archlinux.org/packages/kgtk/) requires gtk2>=2.6. The tarballl also includes a gtk2 folder with ktgk.c file. Doesn't that imply KGtk should work with GKT2 applications like Inkscape? Is there a step we missed?
